app/views/newsfeeds/_post_box.html.erb

```diff
@@ -0,0 +1,81 @@
+<% posts.each do |post| %>
+        <% if post.id %>
+        <div class="post-box">
+        <% if post.user.profile_pic %>
+        <%= image_tag post.user.profile_pic.avatar.url(:comment), alt: "friend's picture", class:  "side-image small-image" %>
+        <% else %>
```

Comment:
I used this same style of logic in my views. Probably should pull it out into a view helper to clean it up